[Bug]: Issue with tests in hop-transform-cratedbbulkloader on Windows

### Apache Hop version?

2.10.0

### Java version?

openjdk version "17.0.11" 2024-04-16

### Operating system

Windows

### What happened?

when you run mvn clean test the following test fails.

[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.apache.hop.pipeline.transforms.cratedbbulkloader.http.CrateDBBulkImportTest
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.686 s <<< FAILURE! -- in org.apache.hop.pipeline.transforms.cratedbbulkloader.http.CrateDBBulkImportTest
[ERROR] org.apache.hop.pipeline.transforms.cratedbbulkloader.http.CrateDBBulkImportTest -- Time elapsed: 0.686 s <<< ERROR!
java.lang.IllegalStateException: Could not find a valid Docker environment. Please see logs and check configuration
        at org.testcontainers.dockerclient.DockerClientProviderStrategy.lambda$getFirstValidStrategy$7(DockerClientProviderStrategy.java:277)
        at java.base/java.util.Optional.orElseThrow(Optional.java:403)
        at org.testcontainers.dockerclient.DockerClientProviderStrategy.getFirstValidStrategy(DockerClientProviderStrategy.java:268)
        at org.testcontainers.DockerClientFactory.getOrInitializeStrategy(DockerClientFactory.java:152)
        at org.testcontainers.DockerClientFactory.client(DockerClientFactory.java:194)
        at org.testcontainers.DockerClientFactory$1.getDockerClient(DockerClientFactory.java:106)
        at com.github.dockerjava.api.DockerClientDelegate.authConfig(DockerClientDelegate.java:109)
        at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:329)
        at org.testcontainers.junit.jupiter.TestcontainersExtension$StoreAdapter.start(TestcontainersExtension.java:280)
        at org.testcontainers.junit.jupiter.TestcontainersExtension$StoreAdapter.access$200(TestcontainersExtension.java:267)
        at org.testcontainers.junit.jupiter.TestcontainersExtension.lambda$null$4(TestcontainersExtension.java:82)
        at org.testcontainers.junit.jupiter.TestcontainersExtension.lambda$startContainers$5(TestcontainersExtension.java:82)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
        at org.testcontainers.junit.jupiter.TestcontainersExtension.startContainers(TestcontainersExtension.java:82)
        at org.testcontainers.junit.jupiter.TestcontainersExtension.beforeAll(TestcontainersExtension.java:56)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)

[INFO] Running org.apache.hop.pipeline.transforms.cratedbbulkloader.http.HttpClientBulkImportResponseTest
OpenJDK 64-Bit Server VM warning: Sharing is only supported for boot loader classes because bootstrap classpath has been appended
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.255 s -- in org.apache.hop.pipeline.transforms.cratedbbulkloader.http.HttpClientBulkImportResponseTest
[INFO] 
[INFO] Results:
[INFO]
[ERROR] Errors: 
[ERROR]   CrateDBBulkImportTest » IllegalState Could not find a valid Docker environment. Please see logs and check configuration
[INFO]
[ERROR] Tests run: 5, Failures: 0, Errors: 1, Skipped: 0
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Hop Orchestration Platform 2.10.0-SNAPSHOT:
[INFO]
[INFO] Hop Orchestration Platform ......................... SUCCESS [  2.511 s]
[INFO] Hop Core ........................................... SUCCESS [ 28.212 s]
[INFO] Hop Engine ......................................... SUCCESS [01:03 min]
[INFO] Hop Engine Beam .................................... SUCCESS [  0.292 s]
[INFO] Hop Libraries ...................................... SUCCESS [  0.045 s]
[INFO] Hop JDBC Libraries ................................. SUCCESS [  0.044 s]
[INFO] Hop GUI ............................................ SUCCESS [  9.350 s]
[INFO] Hop Plugins ........................................ SUCCESS [  0.104 s]
[INFO] Hop Plugins Actions ................................ SUCCESS [  0.059 s]
[INFO] Hop Plugins Actions Abort .......................... SUCCESS [  3.526 s]
[INFO] Hop Plugins Actions Add filenames to result ........ SUCCESS [  3.785 s]
[INFO] Hop Plugins Actions AS/400 Command ................. SUCCESS [  3.573 s]
[INFO] Hop Plugins Actions Check Db connections ........... SUCCESS [  1.844 s]
[INFO] Hop Plugins Actions Check file locked .............. SUCCESS [  1.797 s]
[INFO] Hop Plugins Actions Columns exist in a table ....... SUCCESS [  1.951 s]
[INFO] Hop Plugins Actions Copy files ..................... SUCCESS [  6.533 s]
[INFO] Hop Plugins Actions Copy or Move result filenames .. SUCCESS [  4.389 s]
[INFO] Hop Plugins Actions Create file .................... SUCCESS [  3.832 s]
[INFO] Hop Plugins Actions Create folder .................. SUCCESS [  1.826 s]
[INFO] Hop Plugins Actions Delay .......................... SUCCESS [  4.064 s]
[INFO] Hop Plugins Actions Delete file .................... SUCCESS [  3.984 s]
[INFO] Hop Plugins Actions Delete files ................... SUCCESS [  6.148 s]
[INFO] Hop Plugins Actions Delete folders ................. SUCCESS [  3.655 s]
[INFO] Hop Plugins Actions Delete filenames from result ... SUCCESS [  3.933 s]
[INFO] Hop Plugins Actions Convert file between Windows and Unix SUCCESS [  4.504 s]
[INFO] Hop Plugins Actions JavaScript ..................... SUCCESS [  3.489 s]
[INFO] Hop Plugins Actions Evaluate files metrics ......... SUCCESS [  3.644 s]
[INFO] Hop Plugins Actions Evaluate rows number in a table  SUCCESS [  6.546 s]
[INFO] Hop Plugins Actions File compare ................... SUCCESS [  3.838 s]
[INFO] Hop Plugins Actions File exists .................... SUCCESS [  3.975 s]
[INFO] Hop Plugins Actions Check if files exist ........... SUCCESS [  5.719 s]
[INFO] Hop Plugins Actions Check if a folder is empty ..... SUCCESS [  5.257 s]
[INFO] Hop Plugins Actions Compare folders ................ SUCCESS [  3.801 s]
[INFO] Hop Plugins Actions Get/Put/Delete a file with FTP/SFTP SUCCESS [  8.363 s]
[INFO] Hop Plugins Actions Get mails (POP3/IMAP) .......... SUCCESS [  6.072 s]
[INFO] Hop Plugins Actions HTTP ........................... SUCCESS [  6.603 s]
[INFO] Hop Plugins Actions Mail ........................... SUCCESS [  4.129 s]
[INFO] Hop Plugins Actions Mail validator ................. SUCCESS [  1.863 s]
[INFO] Hop Plugins Technology ............................. SUCCESS [  0.044 s]
[INFO] Hop Plugins Technology Azure ....................... SUCCESS [  3.892 s]
[INFO] Hop Plugins Actions Move files ..................... SUCCESS [  3.928 s]
[INFO] Hop Plugins Actions Display Msgbox Info ............ SUCCESS [  3.535 s]
[INFO] Hop Plugins Actions MS SQL Bulk .................... SUCCESS [  3.794 s]
[INFO] Hop Plugins Actions Bulk load from MySQL into file . SUCCESS [  3.453 s]
[INFO] Hop Plugins Actions Bulk load into MySQL ........... SUCCESS [  3.615 s]
[INFO] Hop Plugins Actions Encrypt/Decrypt/Verify files with PGP SUCCESS [  4.436 s]
[INFO] Hop Plugins Actions Ping a host .................... SUCCESS [  3.510 s]
[INFO] Hop Plugins Actions Pipeline ....................... SUCCESS [  3.054 s]
[INFO] Hop Plugins Actions Repeat ......................... SUCCESS [  0.236 s]
[INFO] Hop Plugins Actions Send Nagios passive check ...... SUCCESS [  3.571 s]
[INFO] Hop Plugins Actions Set variables .................. SUCCESS [  3.334 s]
[INFO] Hop Plugins Actions Shell .......................... SUCCESS [  5.062 s]
[INFO] Hop Plugins Actions Simple evaluation .............. SUCCESS [  1.904 s]
[INFO] Hop Plugins Actions Send SNMP trap ................. SUCCESS [  3.575 s]
[INFO] Hop Plugins Actions SQL ............................ SUCCESS [  0.228 s]
[INFO] Hop Plugins Actions SQL ............................ SUCCESS [  3.485 s]
[INFO] Hop Plugins Actions Success ........................ SUCCESS [  0.224 s]
[INFO] Hop Plugins Actions Table exists ................... SUCCESS [  3.487 s]
[INFO] Hop Plugins Actions Telnet a host .................. SUCCESS [  3.548 s]
[INFO] Hop Plugins Actions Truncate tables ................ SUCCESS [  3.589 s]
[INFO] Hop Plugins Actions Unzip file ..................... SUCCESS [  5.512 s]
[INFO] Hop Plugins Actions Wait for file .................. SUCCESS [  3.615 s]
[INFO] Hop Plugins Actions Wait for SQL ................... SUCCESS [  1.841 s]
[INFO] Hop Plugins Actions Check webservice availability .. SUCCESS [  3.485 s]
[INFO] Hop Plugins Actions Workflow ....................... SUCCESS [  0.224 s]
[INFO] Hop Plugins Actions Write to file .................. SUCCESS [  3.613 s]
[INFO] Hop Plugins Actions Write To Log ................... SUCCESS [  5.608 s]
[INFO] Hop Plugins Transforms ............................. SUCCESS [  0.064 s]
[INFO] Hop Plugins Transforms XML ......................... SUCCESS [ 13.973 s]
[INFO] Hop Plugins Actions XML ............................ SUCCESS [  0.392 s]
[INFO] Hop Plugins Actions Zip file ....................... SUCCESS [  3.720 s]
[INFO] Hop Plugins Databases .............................. SUCCESS [  0.044 s]
[INFO] Hop Plugins Databases Microsoft Access ............. SUCCESS [  1.940 s]
[INFO] Hop Plugins Databases AS400 ........................ SUCCESS [  2.056 s]
[INFO] Hop Plugins Databases InterSystems Cache ........... SUCCESS [  2.349 s]
[INFO] Hop Plugins Databases ClickHouse ................... SUCCESS [  1.822 s]
[INFO] Hop Plugins Databases PostgreSQL ................... SUCCESS [  4.605 s]
[INFO] Hop Plugins Databases CockroachDB .................. SUCCESS [  0.233 s]
[INFO] Hop Plugins Databases CrateDB ...................... SUCCESS [  4.296 s]
[INFO] Hop Plugins Databases DB2 .......................... SUCCESS [  1.847 s]
[INFO] Hop Plugins Databases Apache Derby ................. SUCCESS [  1.847 s]
[INFO] Hop Plugins Databases MySQL ........................ SUCCESS [  5.584 s]
[INFO] Hop Plugins Databases Apache Doris ................. SUCCESS [  0.212 s]
[INFO] Hop Plugins Databases DuckDB ....................... SUCCESS [  0.209 s]
[INFO] Hop Plugins Databases Exasol ....................... SUCCESS [  1.828 s]
[INFO] Hop Plugins Databases Firebird ..................... SUCCESS [  1.873 s]
[INFO] Hop Plugins Databases Generic ...................... SUCCESS [  1.579 s]
[INFO] Hop Plugins Databases Google BigQuery .............. SUCCESS [  1.822 s]
[INFO] Hop Plugins Databases Greenplum .................... SUCCESS [  4.198 s]
[INFO] Hop Plugins Databases H2 ........................... SUCCESS [  1.869 s]
[INFO] Hop Plugins Databases Apache Hive .................. SUCCESS [  5.068 s]
[INFO] Hop Plugins Databases Hypersonic ................... SUCCESS [  1.861 s]
[INFO] Hop Plugins Databases Infobright ................... SUCCESS [  1.902 s]
[INFO] Hop Plugins Databases Informix ..................... SUCCESS [  1.832 s]
[INFO] Hop Plugins Databases Ingres ....................... SUCCESS [  1.959 s]
[INFO] Hop Plugins Databases Interbase .................... SUCCESS [  2.299 s]
[INFO] Hop Plugins Databases InterSystems Iris ............ SUCCESS [  2.114 s]
[INFO] Hop Plugins Databases KingbaseES ................... SUCCESS [  1.941 s]
[INFO] Hop Plugins Databases MariaDB ...................... SUCCESS [  3.979 s]
[INFO] Hop Plugins Databases Monet DB ..................... SUCCESS [  1.942 s]
[INFO] Hop Plugins Databases MS SQL Server ................ SUCCESS [  5.029 s]
[INFO] Hop Plugins Databases MS SQL Server (native) ....... SUCCESS [  1.982 s]
[INFO] Hop Plugins Databases Netteza ...................... SUCCESS [  4.380 s]
[INFO] Hop Plugins Databases Oracle ....................... SUCCESS [  4.953 s]
[INFO] Hop Plugins Databases Oracle RDB ................... SUCCESS [  1.855 s]
[INFO] Hop Plugins Databases Redshift ..................... SUCCESS [  1.532 s]
[INFO] Hop Plugins Databases MaxDB (SAP DB) ............... SUCCESS [  1.849 s]
[INFO] Hop Plugins Databases Snowflake .................... SUCCESS [  1.855 s]
[INFO] Hop Plugins Databases Gupta SQLBase ................ SUCCESS [  1.919 s]
[INFO] Hop Plugins Databases SQLite ....................... SUCCESS [  4.452 s]
[INFO] Hop Plugins Databases Sybase ....................... SUCCESS [  1.952 s]
[INFO] Hop Plugins Databases SybaseIQ ..................... SUCCESS [  1.882 s]
[INFO] Hop Plugins Databases Teradata ..................... SUCCESS [  4.795 s]
[INFO] Hop Plugins Databases UniVerse ..................... SUCCESS [  2.177 s]
[INFO] Hop Plugins Databases Vectorwise ................... SUCCESS [  2.346 s]
[INFO] Hop Plugins Databases Vertica ...................... SUCCESS [  6.229 s]
[INFO] Hop Plugins Transforms Constant .................... SUCCESS [  9.136 s]
[INFO] Hop Plugins Transforms Filter Rows ................. SUCCESS [  4.269 s]
[INFO] Hop Plugins Transforms Group By .................... SUCCESS [  1.815 s]
[INFO] Hop Plugins Transforms Memory Group By ............. SUCCESS [  7.754 s]
[INFO] Hop Plugins Transforms Merge Join .................. SUCCESS [  5.507 s]
[INFO] Hop Plugins Transforms Row Generator ............... SUCCESS [  7.304 s]
[INFO] Hop Plugins Transforms Sort Rows ................... SUCCESS [  7.011 s]
[INFO] Hop Plugins Transforms Stream Lookup ............... SUCCESS [  6.980 s]
[INFO] Hop Plugins Transforms Field Splitter .............. SUCCESS [  7.310 s]
[INFO] Hop Plugins Transforms Switch Case ................. SUCCESS [  6.699 s]
[INFO] Hop Plugins Transforms Unique Rows ................. SUCCESS [  5.623 s]
[INFO] Hop Plugins Engines ................................ SUCCESS [  0.045 s]
[INFO] Hop Plugins Engines Beam ........................... SUCCESS [ 59.978 s]
[INFO] Hop Plugins Miscellaneous .......................... SUCCESS [  0.047 s]
[INFO] Hop Plugins Miscellaneous Asynchronous Web Service . SUCCESS [  0.296 s]
[INFO] Hop Plugins Miscellaneous Debug .................... SUCCESS [  0.280 s]
[INFO] Hop Plugins Miscellaneous Git ...................... SUCCESS [ 13.721 s]
[INFO] Hop Plugins Miscellaneous Import ................... SUCCESS [  0.282 s]
[INFO] Hop Plugins Miscellaneous Passwords ................ SUCCESS [  3.534 s]
[INFO] Hop Plugins Miscellaneous Projects ................. SUCCESS [  0.351 s]
[INFO] Hop Plugins Miscellaneous Reflection ............... SUCCESS [  0.357 s]
[INFO] Hop Static Schema .................................. SUCCESS [  0.227 s]
[INFO] Hop Plugins Miscellaneous Testing .................. SUCCESS [  0.411 s]
[INFO] Hop Plugins Technology Avro ........................ SUCCESS [  1.820 s]
[INFO] Hop Plugins Technology AWS ......................... SUCCESS [  8.858 s]
[INFO] Hop Plugins Technology Cassandra ................... SUCCESS [  8.949 s]
[INFO] Hop Plugins Technology Dropbox ..................... SUCCESS [  0.240 s]
[INFO] Hop Plugins Technology Google ...................... SUCCESS [  0.738 s]
[INFO] Hop Plugins Technology Neo4j ....................... SUCCESS [  4.497 s]
[INFO] Hop Plugins Technology Parquet ..................... SUCCESS [  0.413 s]
[INFO] Hop Plugins Transforms Abort ....................... SUCCESS [  4.471 s]
[INFO] Hop Plugins Transforms Add Sequence ................ SUCCESS [  6.348 s]
[INFO] Hop Plugins Transforms Analytic Query .............. SUCCESS [  4.216 s]
[INFO] Hop Plugins Transforms Append ...................... SUCCESS [  4.053 s]
[INFO] Hop Plugins Transforms Blocking .................... SUCCESS [  1.672 s]
[INFO] Hop Plugins Transforms Block Until Transforms Finish SUCCESS [  1.727 s]
[INFO] Hop Plugins Transforms Calculator .................. SUCCESS [ 17.185 s]
[INFO] Hop Plugins Transforms Change File Encoding ........ SUCCESS [  5.631 s]
[INFO] Hop Plugins Transforms CheckSum .................... SUCCESS [  6.206 s]
[INFO] Hop Plugins Transforms Clone Row ................... SUCCESS [  6.195 s]
[INFO] Hop Plugins Transforms Closure Generator ........... SUCCESS [  5.535 s]
[INFO] Hop Plugins Transforms Coalesce .................... SUCCESS [  5.580 s]
[INFO] Hop Plugins Transforms Column exists ............... SUCCESS [  5.374 s]
[INFO] Hop Plugins Transforms Combination Lookup .......... SUCCESS [  3.618 s]
[INFO] Hop Plugins Transforms Concat Fields ............... SUCCESS [  1.716 s]
[INFO] Hop Plugins Transforms CrateDB bulk loader ......... FAILURE [  4.668 s]
[INFO] Hop Plugins Transforms Credit Card Validator ....... SKIPPED
[INFO] Hop Plugins Transforms Cube Input .................. SKIPPED
[INFO] Hop Plugins Transforms Cube Output ................. SKIPPED
[INFO] Hop Plugins Transforms Database Join ............... SKIPPED
[INFO] Hop Plugins Transforms Database Lookup ............. SKIPPED
[INFO] Hop Plugins Transforms Datagrid .................... SKIPPED
[INFO] Hop Plugins Transforms Database Procedure .......... SKIPPED
[INFO] Hop Plugins Transforms Delay ....................... SKIPPED
[INFO] Hop Plugins Transforms Calculator .................. SKIPPED
[INFO] Hop Plugins Transforms Denormaliser ................ SKIPPED
[INFO] Hop Plugins Transforms Detect Empty Stream ......... SKIPPED
[INFO] Hop Plugins Transforms Detect Last Row ............. SKIPPED
[INFO] Hop Plugins Transforms Dimension Lookup ............ SKIPPED
[INFO] Hop Plugins Transforms Doris Bulk Loader ........... SKIPPED
[INFO] Hop Plugins Transforms Drools ...................... SKIPPED
[INFO] Hop Plugins Transforms Dynamic SQL Row ............. SKIPPED
[INFO] Hop Plugins Transforms EDI 2 XML ................... SKIPPED
[INFO] Hop Plugins Transforms Text File ................... SKIPPED
[INFO] Hop Plugins Transforms Excel ....................... SKIPPED
[INFO] Hop Plugins Transforms Execution Information ....... SKIPPED
[INFO] Hop Plugins Transforms Execute Process ............. SKIPPED
[INFO] Hop Plugins Transforms SQL ......................... SKIPPED
[INFO] Hop Plugins Transforms Execute SQL Row ............. SKIPPED
[INFO] Hop Plugins Transforms Fake ........................ SKIPPED
[INFO] Hop Plugins Transforms Fields Changing Sequence .... SKIPPED
[INFO] Hop Plugins Transforms File Exists ................. SKIPPED
[INFO] Hop Plugins Transforms File Locked ................. SKIPPED
[INFO] Hop Plugins Transforms File Metadata ............... SKIPPED
[INFO] Hop Plugins Transforms Files From Result ........... SKIPPED
[INFO] Hop Plugins Transforms Files To Result ............. SKIPPED
[INFO] Hop Plugins Transforms Row Flattener ............... SKIPPED
[INFO] Hop Plugins Transforms Formula ..................... SKIPPED
[INFO] Hop Plugins Transforms Fuzzy Match ................. SKIPPED
[INFO] Hop Plugins Transforms Get File Names .............. SKIPPED
[INFO] Hop Plugins Transforms Get Files Row Count ......... SKIPPED
[INFO] Hop Plugins Transforms Get Subfolders .............. SKIPPED
[INFO] Hop Plugins Transforms Get Table Names ............. SKIPPED
[INFO] Hop Plugins Transforms Get Variable ................ SKIPPED
[INFO] Hop Plugins Transforms HTTP ........................ SKIPPED
[INFO] Hop Plugins Transforms HTTP Post ................... SKIPPED
[INFO] Hop Plugins Transforms If Null ..................... SKIPPED
[INFO] Hop Plugins Transforms Insert Update ............... SKIPPED
[INFO] Hop Plugins Transforms Janino ...................... SKIPPED
[INFO] Hop Plugins Transforms Load File Input ............. SKIPPED
[INFO] Hop Plugins Transforms Javascript .................. SKIPPED
[INFO] Hop Plugins Transforms JDBC Metadata ............... SKIPPED
[INFO] Hop Plugins Transforms Join Rows ................... SKIPPED
[INFO] Hop Plugins Transforms JSON ........................ SKIPPED
[INFO] Hop Plugins Transforms Pipeline Executor ........... SKIPPED
[INFO] Hop Plugins Transforms Kafka ....................... SKIPPED
[INFO] Hop Plugins Transforms LDAP ........................ SKIPPED
[INFO] Hop Plugins Transforms Mail ........................ SKIPPED
[INFO] Hop Plugins Transforms Mail Input .................. SKIPPED
[INFO] Hop Plugins Transforms Mapping ..................... SKIPPED
[INFO] Hop Plugins Transforms Merge Rows (Diff) ........... SKIPPED
[INFO] Hop Plugins Transforms Metadata Transforms ......... SKIPPED
[INFO] Hop Plugins Transforms Rows From Result ............ SKIPPED
[INFO] Hop Plugins Transforms ETL Metadata Injection ...... SKIPPED
[INFO] Hop Plugins Transforms Metadata Structure of Stream  SKIPPED
[INFO] Hop Plugins Transforms MonetDB Bulk Loader ......... SKIPPED
[INFO] Hop Plugins Transforms MongoDB ..................... SKIPPED
[INFO] Hop Plugins Transforms Multi Merge ................. SKIPPED
[INFO] Hop Plugins Transforms Row Normaliser .............. SKIPPED
[INFO] Hop Plugins Transforms Null If ..................... SKIPPED
[INFO] Hop Plugins Transforms Number Range ................ SKIPPED
[INFO] Hop Plugins Transforms Oracle bulk loader .......... SKIPPED
[INFO] Hop Plugins Transforms PostgreSQL bulk loader ...... SKIPPED
[INFO] Hop Plugins Transforms PGP ......................... SKIPPED
[INFO] Hop Plugins Transforms Process Files ............... SKIPPED
[INFO] Hop Plugins Transforms Property Input .............. SKIPPED
[INFO] Hop Plugins Transforms Property Output ............. SKIPPED
[INFO] Hop Plugins Transforms Random Value ................ SKIPPED
[INFO] Hop Plugins Transforms Regular Expression Evaluation SKIPPED
[INFO] Hop Plugins Transforms Replace In String ........... SKIPPED
[INFO] Hop Plugins Transforms Reservoir Sampling .......... SKIPPED
[INFO] Hop Plugins Transforms REST ........................ SKIPPED
[INFO] Hop Plugins Transforms Rows To Result .............. SKIPPED
[INFO] Hop Plugins Transforms Salesforce .................. SKIPPED
[INFO] Hop Plugins Transforms Sample Rows ................. SKIPPED
[INFO] Hop Plugins Transforms SAS Input ................... SKIPPED
[INFO] Hop Plugins Transforms Schema Mapping .............. SKIPPED
[INFO] Hop Plugins Transforms Script ...................... SKIPPED
[INFO] Hop Plugins Transforms Select Values ............... SKIPPED
[INFO] Hop Plugins Transforms Server Status ............... SKIPPED
[INFO] Hop Plugins Transforms Set Value to a Constant ..... SKIPPED
[INFO] Hop Plugins Transforms Set Value to a Field ........ SKIPPED
[INFO] Hop Plugins Transforms Set Variables ............... SKIPPED
[INFO] Hop Plugins Transforms Snowflake Bulk Loader ....... SKIPPED
[INFO] Hop Plugins Transforms Sorted Merge ................ SKIPPED
[INFO] Hop Plugins Transforms Split Field To Rows ......... SKIPPED
[INFO] Hop Plugins Transforms Splunk ...................... SKIPPED
[INFO] Hop Plugins Transforms SQL File Output ............. SKIPPED
[INFO] Hop Plugins Transforms SSH ......................... SKIPPED
[INFO] Hop Plugins Transforms Standardize Phone Number .... SKIPPED
[INFO] Hop Plugins Transforms Stream Schema Merge ......... SKIPPED
[INFO] Hop Plugins Transforms String Cut .................. SKIPPED
[INFO] Hop Plugins Transforms String Operations ........... SKIPPED
[INFO] Hop Plugins Transforms Switch Case ................. SKIPPED
[INFO] Hop Plugins Transforms System Data ................. SKIPPED
[INFO] Hop Plugins Transforms Table Compare ............... SKIPPED
[INFO] Hop Plugins Transforms Table Exists ................ SKIPPED
[INFO] Hop Plugins Transforms Table Input ................. SKIPPED
[INFO] Hop Plugins Transforms Table Output ................ SKIPPED
[INFO] Hop Plugins Transforms Teradata Fastload Bulk Loader SKIPPED
[INFO] Hop Plugins Transforms Tika ........................ SKIPPED
[INFO] Hop Plugins Transforms Token Replacement ........... SKIPPED
[INFO] Hop Plugins Transforms Unique Rows By Hash Set ..... SKIPPED
[INFO] Hop Plugins Transforms Update ...................... SKIPPED
[INFO] Hop Plugins Transforms Data Validator .............. SKIPPED
[INFO] Hop Plugins Transforms Value Mapper ................ SKIPPED
[INFO] Hop Plugins Transforms Vertica Bulk Loader ......... SKIPPED
[INFO] Hop Plugins Transforms Web Service Available ....... SKIPPED
[INFO] Hop Plugins Transforms Web Services ................ SKIPPED
[INFO] Hop Plugins Transforms Workflow Executor ........... SKIPPED
[INFO] Hop Plugins Transforms Write To Log ................ SKIPPED
[INFO] Hop Plugins Transforms Yaml Input .................. SKIPPED
[INFO] Hop Plugins Transforms Zipfile ..................... SKIPPED
[INFO] Hop Plugins Value Types ............................ SKIPPED
[INFO] Hop Plugins Value Types JSON ....................... SKIPPED
[INFO] Hop GUI (RAP fragment) ............................. SKIPPED
[INFO] Hop GUI (RCP fragment) ............................. SKIPPED
[INFO] Hop REST ........................................... SKIPPED
[INFO] Hop Assemblies ..................................... SKIPPED
[INFO] Hop Assemblies Plugins ............................. SKIPPED
[INFO] Hop Assemblies Static Assets ....................... SKIPPED
[INFO] Hop Client ......................................... SKIPPED
[INFO] Hop Assemblies Debug ............................... SKIPPED
[INFO] Hop Assemblies Web ................................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  11:49 min
[INFO] Finished at: 2024-08-19T15:51:36Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.2.5:test (default-test) on project hop-transform-cratedbbulkloader:
[ERROR]
[ERROR] Please refer to C:\Users\gvdut\IdeaProjects\hop\plugins\transforms\cratedbbulkloader\target\surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :hop-transform-cratedbbulkloader


### Issue Priority

Priority: 3

### Issue Component

Component: Transforms

--- a/plugins/transforms/cratedbbulkloader/src/test/java/org/apache/hop/pipeline/transforms/cratedbbulkloader/http/CrateDBBulkImportTest.java
+++ b/plugins/transforms/cratedbbulkloader/src/test/java/org/apache/hop/pipeline/transforms/cratedbbulkloader/http/CrateDBBulkImportTest.java
@@ -40,7 +40,6 @@ import org.apache.hop.core.row.value.ValueMetaInteger;
 import org.apache.hop.core.row.value.ValueMetaNumber;
 import org.apache.hop.core.row.value.ValueMetaPluginType;
 import org.apache.hop.core.row.value.ValueMetaString;
-import org.apache.hop.databases.cratedb.CrateDBDatabaseMeta;
 import org.apache.hop.pipeline.Pipeline;
 import org.apache.hop.pipeline.PipelineMeta;
 import org.apache.hop.pipeline.engines.local.LocalPipelineEngine;
@@ -60,18 +59,13 @@ import org.testcontainers.cratedb.CrateDBContainer;
 import org.testcontainers.junit.jupiter.Container;
 import org.testcontainers.junit.jupiter.Testcontainers;
 
-@Testcontainers
+@Testcontainers(disabledWithoutDocker = true)
 public class CrateDBBulkImportTest {
-  //
-  //    @ClassRule
-  //    public static RestoreHopEngineEnvironment env = new RestoreHopEngineEnvironment();
 
   @Container
   public static CrateDBContainer crateDBContainer = new CrateDBContainer("crate").withReuse(true);
 
   private static Connection connection;
-
-  private CrateDBDatabaseMeta nativeMeta = new CrateDBDatabaseMeta();
 
   @BeforeAll
   public static void init() throws Exception {

